bigflow/log.py

```diff
@@ -132,8 +134,8 @@ def init_logging(config: LogConfigDict, workflow_id: str):
 
     root = logging.getLogger()
     if not root.handlers:
-        # logs are not configured yet - print to stderr
-        logging.basicConfig(level=log_level)
+        # logs are not configured yet - print to stdout
+        logging.basicConfig(level=log_level, stream=sys.stdout)
```

Comment:
Maybe remove this because for now i am testing it and  we are not sure if this will resolve our problems.

Reply:
done.